### PR TITLE
Deprecate additional-tags file

### PR DESCRIPTION
**Pull request by rcerven**
* CLOUDBLD-74

Signed-off-by: Robert Cerven <rcerven@redhat.com>

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [ ] Pull request has a link to an osbs-docs PR for user documentation updates

### CODE REVIEW

File: osbs/repo_utils.py [hunk 1]
```diff
@@ -239,6 +239,8 @@ def __init__(self, dir_path='', file_name=ADDITIONAL_TAGS_FILE, tags=None):
         self._from_container_yaml = True if tags else False
         self._file_path = os.path.join(dir_path, file_name)
 
+        if os.path.exists(self._file_path):
+            logger.warning('%s file is deprecated, use tags in container.yaml instead', file_name)
```
Comment (athos-ribeiro) [inline osbs/repo_utils.py:243]: Could we be slightly more emphatic here to say ".. is deprecated and will no longer be supported in a future version. Please consider using the tags list in ...."?

**Comment by athos-ribeiro** [comment]
Relates to https://github.com/containerbuildsystem/atomic-reactor/issues/1466

File: osbs/repo_utils.py (hunk 1 above)
Comment (rcerven) [reply osbs/repo_utils.py]: updated

**Comment by rcerven** [comment]
rn updated: Deprecate additional-tags file

**Pull request merged**